### ISSUE TITLE
some space motions don't work

### ISSUE THREAD

**Issue by rococode** [rococode/vimflowy#9]
dt<space> doesn't delete to the next space character despite consuming the dt motion

**Comment by WuTheFWasThat** [comment]
you mean `dt ` with an explicit space?

**Comment by rococode** [comment]
Yeah, I think in general using motions with a space character (single spacebar press) ` ` as the character to jump to doesn't seem to work.

Just tested it and at least these don't seem to work with a space, while using other characters is fine:

`t `,
`f `,
`F `, and
`T `.
